### DISCUSSION TITLE
Welcome to SSBH Editor Discussions!

### DISCUSSION THREAD

**Discussion by ScanMountGoat** [ScanMountGoat/ssbh_editor#75] [Announcements]
Welcome to the SSBH Editor discussions section! Discussions are for topics that aren't suited to issues. Think of this as a forum for anything related to SSBH Editor. If you don't have a GitHub account, you can create one for free following the instructions on GitHub's website. Anyone with an account can make new discussions and post answers to existing discussions. 

Please use discussions to 
  * Ask for help if you're confused about how to use the program or what certain technical terms mean.
  * Discuss potential changes and new features to help shape the future of the application.
  * Share what features you appreciate the most and how SSBH Editor is helping you to create mods.
  * Request topics to add to the WIKI or documentation
  * Get help with understanding the code if you're interested in contributing or using the libraries in one of your own projects.
  * Get clarifications on what an error message means or why a model isn't showing up properly.
  * Help other community members and reply to other people's discussions.
 
Please do not use discussions for
* Asking for help for external applications like Blender or Switch Toolbox.
* Promoting mods, commissions, etc.
* Asking for step by step instructions on how make mods. Ask on the modding discord if you need help, which is linked on https://gamebanana.com/games/6498.

If you want to request a new feature or report a bug, make an issue in [issues](https://github.com/ScanMountGoat/ssbh_editor/issues). If you're unsure if a topic is ok or not, just make the discussion. I'll let you know if it you should ask for help somewhere else or convert the discussion to a GitHub issue as needed.

**Comment by zrksyd** [comment]
For my "attempts to code something but need help", should I put them ehre or in issues?

**Reply by ScanMountGoat** [reply]
I would make a discussion if you just need to ask a question. If you make progress on something, I can always convert it to an issue later.
